symplyphysics/laws/electricity/current_through_element_is_proporional_to_voltage_und_reversly_proportional_to_resistance.py

```diff
@@ -0,0 +1,21 @@
+from symplyphysics import (
```

Comment:
Typo in the file name: proporional -> proportional.

Reply:
I think you can remove `through_element` from the file name. And you can remove the mention of resistance, to make it shorter.